### DISCUSSION TITLE
Sliding window support

### DISCUSSION THREAD

**Discussion by g7ed6e** [LGouellec/streamiz#267] [Ideas]
Hello i would need support for sliding window.
Is it still not planned yet ? Would you be open to add this feature to Streamiz ?

**Comment by LGouellec** [comment]
Hi @g7ed6e ,

This feature is not delivered yet unfortunately but we can track this feature request and try to release it in the next release.

Can you open a GH issue to track that ?

**Reply by g7ed6e** [reply]
Hi @LGouellec, 
Thanks for quick reply. Please let me know if you need some help for this.
